cms/server/contest/templates/overview.html

```diff
@@ -60,56 +78,56 @@ <h2>{{ _("General information") }}</h2>
 {% if tokens_contest != 0 and tokens_tasks != 0 %}
     {% if tokens_contest == 2 and tokens_tasks == 2 %}
         <p>
-        {{ _("You have an infinite number of tokens.") }}
+        {% trans %}You have an infinite number of tokens.{% endtrans %}
         </p>
 
         <p>
-        {{ _("You can see the detailed result of a submission by using a token on it.") }}
-        {{ _("Your score for each task will be the maximum among the tokened submissions and the last one.") }}
+        {% trans %}You can see the detailed result of a submission by using a token on it.{% endtrans %}
+        {% trans %}Your score for each task will be the maximum among the tokened submissions and the last one.{% endtrans %}
         </p>
     {% elif tokens_contest == 2 %}
         <p>
-        {{ _("You have a distinct set of tokens for each task.") }}
-        {{ _("You can find the rules for the %(type_pl)s on each task's description page.") % {"type_pl": _("tokens")} }}
+        {% trans %}You have a distinct set of tokens for each task.{% endtrans %}
+        {% trans type_pl=_("tokens") %}You can find the rules for the {{ type_pl }} on each task's description page.{% endtrans %}
```

Comment:
Something weird going on here.

Reply:
Yes, but shouldn't you get rid of _?

Reply:
Jinja's i18n extension provides the `_` symbol as an alias to the `gettext` function (that we provide, in the context). Hence this still works. (In fact, this whole commit wasn't technically necessary, as `{{ _("foo") }}` is pretty much the same as `{% trans %}foo{% endtrans %}`; the latter one is more idiomatic though).